app/services/paginated_comments.rb

```diff
@@ -0,0 +1,34 @@
+class PaginatedComments
+  DEFAULT_LIMIT = 10
+
+  def initialize(comments_relation:, params:, direction: :backwards)
+    @params = params
```

Comment:
now that i am thinking about this, we don't need whole params hash `here`, maybe we should assign `limit` and `position` in constructor and check instance variable in getter methods, something like this:

```ruby
def initialize
  @limit    = params[:limit].to_i
  @position = params[:position]
end

def limit
  return DEFAULT_LIMIT if @limit.zero?
  @limit
end

...
```